beanbag_docutils/sphinx/ext/github.py

```diff
@@ -228,6 +228,8 @@ def github_linkcode_resolve(domain, info, github_org_id, github_repo_id,
 
     # Get the branch/tag/commit to link to.
     ref = _get_git_doc_ref(branch) or branch
+    if isinstance(ref, bytes):
```

Comment:
I'm not sure if this `isinstance()` check is necessary - could there be cases where `ref` is a regular Python string?

Reply:
Moved this to `_get_git_doc_ref()`.